src/client/src/rt-platforms/getPlatformAsync.ts

```diff
@@ -45,8 +50,16 @@ export const getPlatformAsync = async () => {
   }
 
   if (isOpenFin) {
-    console.info('Using OpenFin API')
-    const { OpenFin } = await getOpenFinPlatform()
+    let appInfo = await window.fin.Application.getCurrentSync().getInfo()
+
+    if (appInfo.initialOptions.isPlatformController) {
+      console.info('Using OpenFin Platform API')
+      const { OpenFinPlatform } = await getOpenFinPlatform()
+      return new OpenFinPlatform()
+    }
+
+    console.info('Using OpenFin Legacy API')
+    const { OpenFin } = await getOpenFin()
```

Comment:
This is apparently how you distinguish between running under the new Platform API and the legacy API.